CIP-calidus-key/README.md

```diff
@@ -0,0 +1,373 @@
+
+**Fields**
+
+**0: Witness Type Identifier**
+
+The Witness Type identifier is optional for a COSE Witness and is assumed to be
+a value of `0`.
+
+**1: Witness Header or Public Key Identifier**
+
+The `1` key is used to help identify the public key in the case of a simple
+signature witness or provides the COSE Signature Headers in the case of a COSE
+Witness (CIP-0008/CIP-0030).
+
+**2: Witness Signature Payload**
+
+The `2` key is used to contain the actual signature payload for the witness. In
+the case of the COSE signing this will be an array of elements used in the
+validation process. In the case of a simple Ed25519 key signature, this will be
+a simple hex-encoded CBOR witness signature.
+
+> **Note:** COSE Signing as described in CIP-0008 uses a boolean value as the
+> second entry of the COSE Payload to indicate whether the payload was hashed
+> by the signing function before signing. Because Cardano on-chain metadata
+> does not support boolean values, this must be converted to/from a binary (0 or 
+> 1\) value when encoding or decoding the payload.
+> 
+> By default, the COSE Sign1 Payload contains an entry which is an object (map)
+> containing a `hashed` key and a boolean `true/false` value. This entire object
+> should be replaced with a simple `1` or `0` value representing the `true/false`
+> value of the `hashed` key.
+> 
+> Example:
+> `{"hashed": true}` becomes `1` and `{"hashed": false}` becomes `0`. In most
+> cases it will also be required to convert from the `1` or `0` value stored in
+> entry 2 of the COSE Sign1 Payload array back to object notation before
+> validating.
+
+## Rationale: how does this CIP achieve its goals?
+
+This CIP was born out of a desire to allow SPOs to routinely identify themselves
+to third-party services such as: voting platforms, social media, governance 
+without risking or compromising the keys used in the actual operation and setup
+of the stake pool.
+
+Thus, the concept of an authorized "hot key" (the Calidus Key) was born as a
+solution to this authentication issue. Given that [CIP-0088] had already been
+written and was extensible to support this registration with minimal additional
+support effort, extending it was chosen in favor of other solutions.
+
+## Path to Active
+
+### Acceptance Criteria
+
+This CIP should be considered for `Active` status once a substantial amount of
+ecosystem tooling supports it and SPOs as well as third-party applications have
+shown an interest in using it as a method of authentication and validation.
+
+* Ecosystem Tooling
+  * [x] Cardano Signer
+  * [x] pg_cardano
+  * [x] cardano-hw-cli support
+  * [ ] Cardano Koios 
+    * currently testing in Preview on Branch 1.3.2
+  * [ ] Blockfrost
+  * [ ] CN Tools
+  * [ ] SPO Scripts
+* Wallets
+  * [ ] Eternl
+* Applications
+  * [ ] CExplorer
+  * [ ] CardanoScan
+  * [ ] AdaStat
+  * [ ] PoolTool.io
+  * [ ] DripDropz
+* Registered Pools
+  * [ ] _x_ of _y_ SPOs have registered
```

Comment:
@Crypto2099 @gitmachtl is there a quick Blockfrost query we could use to determine `x`?  (and maybe even `y`, for that matter?)

Reply:
Looks like we have one through Koios as per @gitmachtl https://github.com/cardano-foundation/CIPs/pull/999#issuecomment-2720539248 (apologies for my autistic query; feel free to correct with something properly semantic):
```
$ curl -s "https://api.koios.rest/api/beta/pool_calidus_keys" | jq | grep '"pool_status": "registered"' | wc -l
4
```
@Crypto2099 @Ryun1 @perturbing can we make an operating assumption about `y` or is there a quick & dirty way of determining that as well (which properly factors out multi-pool operators)?  cc @Balance-Analytics @ccgarant

Reply:
you can do:

```
curl -s "https://api.koios.rest/api/beta/pool_calidus_keys?pool_status=eq.registered" | jq -r "length"
4
```

```
curl -s "https://preview.koios.rest/api/beta/pool_calidus_keys?pool_status=eq.registered" | jq -r "length"
6
```

but i am generally not sure if a metric x out of y pools must be used as an acceptance criteria.

Reply:
@Crypto2099 that's the way you had posted it (`x` out of `y`); are there any other measures of adoption we could consider along with this one?